themes/src/main/resources-community/theme/base/account/messages/messages_de.properties

```diff
@@ -339,3 +338,21 @@ access-denied-when-idp-auth=Zugriff verweigert bei Authentifizierung mit {0}
 organizationScopeConsentText=Organisation
 updateReadOnlyAttributesRejectedMessage=Aktualisierung eines schreibgeschützten Attributs abgelehnt
 invalidPasswordNotContainsUsernameMessage=Ungültiges Passwort: Darf den Benutzernamen nicht enthalten.
+error-user-attribute-required=Bitte geben Sie Attribut {0} an.
+error-invalid-value=Ungültiger Wert.
+error-empty=Bitte geben Sie einen Wert an.
+error-invalid-length=Das Attribut {0} muss eine Länge zwischen {1} und {2} haben.
+error-invalid-length-too-short=Das Attribut {0} muss mindestens die Länge {1} haben.
+error-number-out-of-range-too-small=Das Attribut {0} muss mindestens den Wert {1} haben.
+error-number-out-of-range-too-big=Das Attribut {0} darf maximal den Wert {2} haben.
+error-pattern-no-match=Ungültiger Wert.
+error-invalid-uri-scheme=Ungültiges URL-Schema.
+error-invalid-date=Ungültiges Datum.
+error-user-attribute-read-only=Das Feld {0} ist read only.
```

Comment:
Could you please try to find a translation for "read only" as well?

Reply:
Then "schreibgeschützt"? Or something like "Auf das Feld {0} haben Sie nur lesenden Zugriff"

Reply:
I'd think I'd go for "schreibgeschützt".

Reply:
Fixed.